trunk/SUAVE/Attributes/Atmospheres/Earth/US_Standard_1976.py

```diff
@@ -56,30 +60,16 @@ def compute_values(self,altitude,type="all"):
 

```

Comment:
this function is duplicating Analyses.Atmospheric.US_1976.  This is an Attribute class so it shouldn't have this kind of class method